### PR TITLE
New Relic support, conditionally-installed

### PR DESCRIPTION
**Pull request by scottrigby**
Here's an initial stab at New Relic support. Untested. CC @cweagans

### CODE REVIEW

**Comment by oxyc** [comment]
You also need to add an entry to https://github.com/geerlingguy/drupal-vm/blob/master/provisioning/requirements.yml

**Comment by cweagans** [comment]
I'd also suggest adding a documentation page for how to get this set up, ideally including some information on adding `php_value newrelic.appname "Whatever"` to the vhost to be able to set the New Relic app name on a per-site basis.

File: example.config.yml [hunk 1]
```diff
@@ -168,6 +169,10 @@ installed_extras:
 # Add any extra apt or yum packages you would like installed.
 extra_packages: []
 
+# `newrelic` must be in installed_extras for this to work.
+# See all vars: https://github.com/weareinteractive/ansible-newrelic#variables
+newrelic_license_key: yourkey
```
Comment (geerlingguy) [inline example.config.yml]: I'd rather leave this out by default, and have a new docs page under 'Extra software and setup' that describes how to enable New Relic support. I have a feeling that the number of people who will use New Relic with Drupal VM is fairly small (percentage-wise), so I'd rather leave these three lines out of the example config.yml

File: provisioning/playbook.yml [hunk 2]
```diff
@@ -43,6 +43,7 @@
     - { role: geerlingguy.pimpmylog, when: '"pimpmylog" in installed_extras' }
     - { role: geerlingguy.daemonize, when: '"mailhog" in installed_extras' }
     - { role: geerlingguy.mailhog, when: '"mailhog" in installed_extras' }
+    - { role: franklinkim.newrelic, when: '"newrelic" in installed_extras' }
```
Comment (geerlingguy) [inline provisioning/playbook.yml]: As @oxyc mentioned, we need to add the role to the `requirements.yml` file as well.

Comment (scottrigby) [inline provisioning/playbook.yml]: OK Added in f3a465d.

File: example.config.yml (hunk 1 above)
Comment (scottrigby) [inline example.config.yml]: @geerlingguy OK, I moved this to docs/extras in d72058f.

**Comment by scottrigby** [comment]
@cweagans  OK, check out `docs/extras/newrelic.md` (in d72058f). Is there anything else you think is needed for documentation?

**Comment by cweagans** [comment]
:+1: from me. Looks like feedback has been taken into account.

**Comment by cweagans** [comment]
(For the record, the app name per vhost thing is a nice-to-have, IMO. I don't think many people will use that, and Pubstack doesn't have it either)

**Comment by geerlingguy** [comment]
Thanks, everyone!

**Pull request merged**